examples/webgl_loader_gltf.html

```diff
@@ -89,14 +155,20 @@
 
 						if ( child.isMesh ) {
 
-							child.material.envMap = envMap;
+							child.material.environment = new THREE.CubeTextureNode( envMap );
+
+							child.material.build();
```

Comment:
A visual node editor that would generate the GLSL would expose "inputs". Rather than having `node( type input)` after compiling it should just be `type input`, if that makes sense? It feels weird to have to rebuild it, but this is basically in lieu of `needsUpdate`?

Reply:
Sorry I don't think I quite follow your question, but yes, `.build()` is analogous to `.needsUpdate` here.